test/Continuity.lean

```diff
@@ -0,0 +1,22 @@
+import Mathlib.Topology.Basic
```

Comment:
I think it's worth a note in a comment that this is not a port of https://github.com/leanprover-community/mathlib/blob/master/test/continuity.lean (yet)

Reply:
Good point, thanks. I've added a porting note.